terraform/shared/database/main.tf

```diff
@@ -0,0 +1,23 @@
+###
+# Target space/org
+###
+
+data "cloudfoundry_space" "space" {
+  org_name = var.cf_org_name
+  name     = var.cf_space_name
+}
+
+###
+# RDS instance
+###
+
+data "cloudfoundry_service" "rds" {
+  name = "aws-rds"
+}
+
+resource "cloudfoundry_service_instance" "rds" {
```

Comment:
[musing] i think based on the above documentation this is supposed to create a service instance if it doesn't exist. did you try it without first creating the service instance manually? sorry if this is a silly question!

Reply:
It seems to work for me when I destroy all the services!  In fact, it complains rather bitterly if it didn't know a service existed when it starts to run for the first time, as I learned when I had manually created an rds instance to make sure I understood how to, then tried to run this terraform code.  I don't think it's a silly question!